src/include/storage/tile_group_header.h

```diff
@@ -228,6 +234,24 @@ class TileGroupHeader : public Printable {
                                         transaction_id);
   }
 
+  /*
+  The following method use Compare and Swap to set the tilegroup's
+  immutable flag to be true. 
+  */
```

Comment:
I'll be the first to point this out .... This doesn't follow the JavaDoc-style formatting guidelines.